documents/kubernetes/README.md

```diff
@@ -0,0 +1,18 @@
+# Kubernetes Deployment Scripts
+
+The CV Manager supports being hosted within a Kubernetes cluster which allows for better stability, smoother deployments and performance scaling. The YAML files within this directory provide a starting point to incorporate the CV Manager into your own Helm deployments in any local or cloud based Kubernetes environment.
+

```

Comment:
Is it worth making a note here that the example YAML files are currently set up to use GCP?

Reply:
Addressed